### PR TITLE
add console.log for recruitment

### LINKED ISSUE #305: Add a console.log for recruitment
A lot of companies will put a console log message on their sites, as a little easter egg for anyone who opens the console. We should do this too and refer people to our github and discord. 

Add something like this to a new script tag in [index.html](https://github.com/icssc-projects/AntAlmanac/blob/main/public/index.html):
```
console.log("Interested in improving AntAlmanac?\n - Checkout the project on GitHub: https://github.com/icssc-project/antalmanac\n - Join our Discord: https://discord.gg/GzF76D7UhY\n - Leave feedback: https://antalmanac.com/feedback")
```

Feel free to get creative with the formatting/content! There's lots of cool examples online of how to do this: https://thenewstack.io/tutorial-getting-creative-with-console-statements/

### PR DESCRIPTION
**Pull request by wwesleym**
## Summary
Added console.log for recruitment.
image

## Test Plan
- Verified that the change showed up after opening AntAlmanac and the console

## Issues
- Closes #305

### CODE REVIEW

File: public/index.html [hunk 1]
```diff
@@ -71,6 +71,14 @@
         <title>AntAlmanac</title>
         <!-- Global site tag (gtag.js) - Google Analytics -->
 
+        <!-- console.log for recruitment -->
+        <script>
+            console.log(
+                '%cInterested in improving AntAlmanac?\n - Checkout the project on GitHub: https://github.com/icssc-project/antalmanac\n - Join our Discord: https://discord.gg/GzF76D7UhY\n - Leave feedback: https://antalmanac.com/feedback',
+                'color: red; font-size: 25px; font-family: Comic Sans'
```
Comment (ChaseC99) [inline public/index.html]: Can you do the following?
- change the color to be the same color as AntAlmanac's navbar (you'll have to look it up in the code)
- remove the comic sans font family (it doesn't look like it renders anyways)

**Review by ChaseC99 — CHANGES_REQUESTED** [review]

File: public/index.html (hunk 1 above)
Comment (wwesleym) [reply public/index.html]: image
like this?

Comment (ChaseC99) [reply public/index.html]: Looks great!

**Review by ChaseC99 — APPROVED** [review]
LGTM! Congrats on your first AntAlmanac contribution 🎉

**Pull request merged**